src/elements/content-sidebar/Sidebar.js

```diff
@@ -117,6 +129,36 @@ class Sidebar extends React.Component<Props, State> {
         history.push(`${history.location.pathname}/versions`);
     };
 
+    /**
+     * Getter/setter for sidebar forced state
+     *
+     * @param isOpen - Optionally set the sidebar to open/closed
+     * @returns {boolean} - The sidebar open/closed state
+     */
+    isForced(isOpen?: boolean): ?boolean {
+        if (isOpen !== undefined) {
+            this.store.setItem(SIDEBAR_FORCE_KEY, isOpen ? SIDEBAR_FORCE_VALUE_OPEN : SIDEBAR_FORCE_VALUE_CLOSED);
+        }
+
+        return this.store.getItem(SIDEBAR_FORCE_KEY);
+    }
+
+    /**
+     * Getter for whether the sidebar has been forced open
+     * @returns {boolean} - True if the sidebar has been forced open
+     */
+    isForcedOpen() {
+        return this.isForced() !== SIDEBAR_FORCE_VALUE_CLOSED;
```

Comment:
isForced returns a boolean, but here we're comparing to the string `closed` so this will never be true?

Reply:
Updated the types to be more clear.